### PR TITLE
Sprint9 165 wvr list component

### LINKED ISSUE #165: Weaver Components should have a list component
Weavr components should provide a single list component which can display listed data in various styles, including: ordered, bulleted, description, un-styled, custom bullet  and custom background list items (list groups). 

https://getbootstrap.com/docs/4.0/content/reboot/#lists

https://getbootstrap.com/docs/4.0/components/list-group/

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #165

### CODE REVIEW

File: projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.scss [hunk 1]
```diff
@@ -0,0 +1,201 @@
+@import "../../shared/styles/host-styles.scss";
+
+:host {
+
+  ::ng-deep {
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.scss:5]: Could this be moved below variable declarations

File: projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  @Input() context: 'primary' | 'secondary' | 'success' | 'danger' | 'warning' | 'info' | 'light' | 'dark';
+
+  @Input() customContentItemHeading: string;
+
+  @Input() cusomContentHeadingSmallText: string;
+
+  @Input() customContentSmallText: string;
+
+  // displayActiveState: boolean = false
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.ts:24]: Does this need to be here?

File: projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.scss (hunk 1 above)
Comment (rmathew1011) [reply projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.scss:5]: Done

File: projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.ts (hunk 2 above)
Comment (rmathew1011) [reply projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.ts:24]: Removed line

**Pull request merged**